templates/settings.yml.erb

```diff
@@ -4,6 +4,8 @@
 
 :settings_directory: <%= scope.lookupvar("foreman_proxy::etc") -%>/foreman-proxy/settings.d
 
+:instance_uuid: <%= scope.lookupvar("foreman_proxy::instance_id") %>
```

Comment:
Reviewing this, I do wonder if we should also configure the Smart Proxy with the signature just like the fact. If we place it in `settings.yml` then a dynamic fact could also extract it. It also means Foreman can validate it if desired.

Reply:
We could, but introducing a setting that most likely, unless we decide we do not trust a proxy, wouldn't be used felt wrong